airbyte-ci/connectors/pipelines/pipelines/dagger/actions/secrets.py

```diff
@@ -136,21 +136,34 @@ async def mounted_connector_secrets(context: PipelineContext, secret_directory_p
         # Special case for local development.
         # Query dagger for the contents of the secrets and mount these strings as files in the container.
         contents = {}
-        for secret_file_name, secret in context.connector_secrets.items():
-            contents[secret_file_name] = await secret.plaintext()
+        if secret_directory_path:
```

Comment:
According to the type annotation `secret_directory_path` is not optional.
Did you find/introduce calls to `mounted_connector_secrets` with None/False `secret_director_path`?
If yes, please change the type annotation to `Optional[str]`

Reply:
Done